code/js/modules/Sitelist.js

```diff
@@ -160,7 +160,8 @@
       "xiami": { name: "Xiami", url: "http://www.xiami.com" },
       "youtube": { name: "YouTube", url: "http://www.youtube.com" },
       "youzeek": { name: "YOUZEEK", url: "http://www.youzeek.com" },
-      "zonga": { name: "Zonga", url: "http://asculta.zonga.ro", controller: "ZongaController.js" }
+      "zonga": { name: "Zonga", url: "http://asculta.zonga.ro", controller: "ZongaController.js" },
+      "udemy": { name: "Udemy", url: "https://www.udemy.com/", controller: "UdemyController.js" }
```

Comment:
Agreed, I've just read from `twitch` and beyond and they weren't alphabetic ordered, so I thought it was to avoid conflicts.

My bad. I'm happy to re-order if you'd like to.